en_US/data-integration/data-bridge-gcp-pubsub.md

```diff
@@ -110,35 +110,38 @@ You can continue to create rules to specify the data to be saved into GCP PubSub
 
 2. Click **Create** on the top right corner of the page.
 
-3. Input `my_rule` as the rule ID, and set the rules in the **SQL Editor**. Here we want to save the MQTT messages under topic `t/#`  to GCP PubSub, we can use the SQL syntax below. 
+3. Input `my_rule` as the rule ID.
 
-   Note: If you want to specify your own SQL syntax, make sure that you have included all fields required by the data bridge in the `SELECT` part.
+3. Set the rules in the **SQL Editor**. Here we want to save the MQTT messages under topic `/devices/+/events`  to GCP PubSub, we can use the SQL syntax below. 
+
+   Note: If you want to specify your own SQL syntax, make sure that the `SELECT` part includes all fields required by the payload template in the data bridge.
    
    ```sql
    SELECT
      *
    FROM
-     "t/#"
+     "/devices/+/events"
    ```
 
-4. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list and then select the data bridge you just created under **Data Bridge**. Then click the **Add** button.
+5. Click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list, and then select the data bridge you just created under **Data Bridge**. Then click the **Add** button.
+
 4. Click **Create** at the page bottom to finish the creation.
 
-Now a rule to forward data to GCP PubSub via a GCP PubSub bridge is created. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/#`  are sent and saved to GCP PubSub after parsing by rule  `my_rule`.
+Now a rule to forward data to GCP PubSub via a GCP PubSub bridge is created. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/devices/+/events`  are sent and saved to GCP PubSub after parsing by rule  `my_rule`.
```

Comment:
```suggestion
Now a rule to forward data to GCP PubSub via a GCP PubSub bridge is created. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `/devices/+/events` are sent and saved to GCP PubSub after parsing by rule `my_rule`.
```